app/index.js

```diff
@@ -35,7 +35,7 @@ module.exports = class extends yeoman.Base {
 			name: 'moduleName',
 			message: 'What do you want to name your module?',
 			default: _s.slugify(this.appname),
-			filter: x => moduleName.slugify(x)
+			filter: x => _s.slugify(x)
```

Comment:
The correct fix would be to fix `moduleName.slugify()`. It's used there for a reason.